android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/JsonFormMLSAssetGenerator.java

```diff
@@ -351,7 +351,7 @@ private static JsonFormInteractor getJsonFormInteractor() throws Exception {
                 ? "com.vijay.jsonwizard.interactors.JsonFormInteractor" : jsonFormInteractorName;
         Class<?> clazz = Class.forName(jsonFormInteractorName);
         Method factoryMethod = clazz.getDeclaredMethod("getInstance");
-        return (JsonFormInteractor) factoryMethod.invoke(null, null);
+        return (JsonFormInteractor) factoryMethod.invoke(null,  (Object[]) null);
```

Comment:
Any reason we are casting this to an Object?

Reply:
Unit tests failing with TypeCastException for the class on this branch.